backend/config/initializers/locale.rb

```diff
@@ -0,0 +1,2 @@
+
+I18n.available_locales = [:en, :"en-PH"]
```

Comment:
Please move this in `config/application.rb`. You can see the guide [here](https://github.com/rails/rails/blob/df94b863c2ff8f1bcf12e36ed8fc1419292668e7/guides/source/configuring.md#configuring-i18n)